openssl/src/pkcs12.rs

```diff
@@ -269,7 +271,10 @@ impl Pkcs12Builder {
                 cvt(ffi::PKCS12_set_mac(
                     pkcs12.as_ptr(),
                     pass,
-                    -1,
+                    // -1 could be passed in for the password length in OpenSSL,
+                    // but AWS-LC does not support that at this time.
+                    // We know the length so explicitly provide it.
```

Comment:
Personally I don't think we have to justify why we're passing the length. (It's useful to this diff reviewer, but not really to future readers of the code.)